website_mrp_bom/views/website_sale_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <template id="mrp_bom_website_sale_details" name="MRP BoM Website Sale Details" inherit_id="website_sale.product">
+        <xpath expr="//a[@id='add_to_cart']" position="after">
+            <t t-if="product.bom_ids">
+               <a id="mrp_bom" class="btn btn-primary btn-lg mt8" t-attf-href="/mrp_bom/#{ slug(product.bom_ids[0]) }">See Components</a>
```

Comment:
don't you have a website url? If not I'd add compute url on the product like "bom_website_url"

Reply:
View removed due to website_sale is not longer a dependency.